docs/creating-components.md

```diff
@@ -156,67 +156,56 @@ const CountableDiv = Countable('div');
 
-However, we do think this is an important feature for the framework and we will continue to roll it out
-throughout and improve the overall developer experience around it. Here's a short example to whet
-your appetite:
+How about an example to make this more clear. Let's customize the background color of a Moonstone [`Button`](../../../modules/moonstone/Button/). `Button` exposes several classes for customization: 'button', 'bg', 'small', and 'selected', and in this case we're interested in 'button' and 'bg'.  In our customized component LESS file, the following should do the trick:
 
 ```
-import factory from '@enact/core/factory';
-import kind from '@enact/core/kind';
-import React from 'react';
+// CustomButton.less
+//
+@import '~@enact/moonstone/styles/skin.less';
+
+.button {
+	.applySkins({
+		.bg {
+			background-color: orange;
+		}
+	});
+}
+```
 
-import componentCss from './Button.less';
+*The `.applySkins` is added here because Moonstone uses our skinning system too, which is in charge of applying colors independent from measurements, layout, and metrics.*
 
-const ButtonFactory = factory({css: componentCss}, ({css}) => {
-	return kind({
-		name: 'Button',
+Then, in our component we'll just apply the imported LESS file to the component with the `css` property.
 
-		// Since 'button' will be resolved against the combined `css` map, it can be overridden too
-		styles: {
-			css,
-			className: 'button'
-		},
+```
+import React from 'react';
+import kind from '@enact/core/kind';
+import Button from '@enact/moonstone/Button';
 
-		// Component authors can also prevent overrides by using their css map directly as is done
-		// with the `inner` class below
-		render: ({children, ...rest}) => (
-			<button {...rest}>
-				<div className={componentCss.inner}>
-					{children}
-				</div>
-			</button>
-		)
-	});
+import css from './CustomButton.less';
+
+const CustomButton = kind({
+	name: 'CustomizedButton',
+
+	render: ({children, ...rest}) => (
+		<Button {...rest} css={css}>{children}</Button>
+	)
 });
-```
 
+export default CustomButton;
 ```
-// If `buttonCss` includes a `button` class, it will be appended to the `button` class of the
-// `Button` component.
-import buttonCss from './CustomButton.less';
-CustomizedButton = ButtonFactory({css: buttonCss});
 
-<CustomizedButton />
-```
+Now, all we do in our app is import this `CustomButton` like any other, and it will be styled with our custom styling.
+
+For more details and advanced theming features and recommendations, see our [Themeing Guide](somewhere). **NOTE: Actually link this somewhere**
```

Comment:
Can you comment this out until we have it?

Reply:
This commit is the first commit to verify that the approach is sound. Before this PR is submitted, I'll be adding the Themeing Guide, and removing the authoring notes, which remind me where I need to go back and put links or other tidying details.